src/components/input/input.html

```diff
@@ -0,0 +1,29 @@
+<div class="md-wrapper" (click)="input.focus()">
+  <div class="md-flex">
+    <prefix><ng-content select="[md-prefix]"></ng-content></prefix>
+
+    <label [attr.for]="'md-input-' + _inputId"
```

Comment:
Store the entire ID on the class instead of doing the string concat twice in the template?